packages/controller-utils/src/types.ts

```diff
@@ -27,6 +27,13 @@ export enum NetworksChainId {
   rpc = '',
 }
 
+export const NetworkIds = {
+  mainnet: '1',
+  goerli: '5',
+  localhost: '1337',
```

Comment:
The `localhost` entry here was probably copied from the `NetworksChainId` constant, which has a `localhost` entry with the chain ID hard-coded as 1337. This was not used for custom RPC entries that happened to use a `localhost` RPC URL; it was used solely for a built-in network meant for enabling e2e tests using Ganache.

This `localhost` config has been dead code literally since this repository was first created though, and we plan to delete it soon as part of the multichain work. Also it's not used here anyway. Agreed that it can be removed.

Reply:
>is there a way to build this without introducing a hardcoded network map in this module (considering that a future change will make this chain-agnostic)?

Do you mean that this constant doesn't belong in `controller-utils`? Or that we shouldn't be using the `ethereum-ens-network` package at all? Not entirely sure I follow.

For context, the scope of this work is to merge the extension ENS controller with the core ENS controller, so that we can delete the extension one. Current the extension uses the `ethereum-ens-network` package, which provides a map of ENS contracts keyed by network IDs. It does seem like the shortest path would be to migrate it as-is for now.